fetcher/fetcher/common.py

```diff
@@ -217,10 +227,53 @@ def gcs_key(self) -> str:
         return f"{hive_table}/{hive_str}/{self.filename}"
 
 
-class FetchedRecord(BaseModel):
+class ParseOutcomeMetadata(BaseModel):
+    hash: str
+
+
+class ParseOutcome(BaseModel):
     file: RawFetchedFile
-    record: Dict[str, Any]
-    line_number: Optional[int]
+    metadata: ParseOutcomeMetadata
+    success: bool
+    exception: Optional[Exception]
+
+    class Config:
+        arbitrary_types_allowed = True
+        json_encoders = {
+            Exception: str,
+            pendulum.DateTime: lambda ts: ts.to_iso8601_string(),
+        }
+
+
+class FeedTypeHourOutcomes(BaseModel):
+    bucket: ClassVar[str] = PARSED_BUCKET
+    partitions: ClassVar[List[str]] = ["dt"]
+    feed_type: FeedType
+    hour: pendulum.DateTime
+
+    @property
+    def table(self) -> str:
+        return f"{self.feed_type.value}__outcomes"
```

Comment:
ah yep good point; I think we've discussed before that parsing + validation could really be part of the same job potentially (to avoid extra network traffic) but I don't think it's actually a good idea so I will separate